### PR TITLE
WF-64205-Add UG for entire pages, first page and specific range of pages in Word to image conversion

### PR DESCRIPTION
**Pull request by Suriya-Balamurugan**
Kindly review the below merge request and let me know your feedback.

<table>
<tr>
<td>
Task Link
</td>
<td>
<a href="https://syncfusion.atlassian.net/browse/WF-64205">https://syncfusion.atlassian.net/browse/WF-64205</a>
</td>
</tr>
<tr>
<td>
Description
</td>
<td>
Add UG document for all the overloads in Word to image conversion.
</td>
</tr>
<tr>
<td>
Solution
</td>
<td>
Added code snippet in Word to image page in all platforms. 

- Convert the entire Word to images
- Convert first page of Word to image
- Convert a specific range of pages in Word to an image

</td>
</tr>
<tr>
<td>
Content review confirmation
</td>
<td>
 Not yet
</td>
</tr>
</table>

### CODE REVIEW

File: File-Formats/DocIO/word-to-image.md [hunk 1]
```diff
@@ -137,6 +138,203 @@ using (Stream docStream = typeof(App).GetTypeInfo().Assembly.GetManifestResource
...
+    wordDocument.ChartToImageConverter.ScalingMode = ScalingMode.Normal
+    'Convert the first page of the Word document into an image.
+    Dim image As Image = wordDocument.RenderAsImages(0, ImageType.Bitmap)
+    'Save the image as jpeg.
+    image.Save("WordToImage.jpeg", ImageFormat.Jpeg)
+End Using
+{% endhighlight %}
+
+{% highlight c# tabtitle="UWP" %}
+//DocIO supports Word to image conversion in Windows Forms, WPF, ASP.NET and ASP.NET MVC platform alone
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/word-to-image.md:178]: //DocIO supports Word to image conversion in UWP application using DocIORenderer.

Check in UWP and add code snippet using DocIORenderer

File: File-Formats/DocIO/word-to-image.md [hunk 2]
```diff
@@ -137,6 +138,203 @@ using (Stream docStream = typeof(App).GetTypeInfo().Assembly.GetManifestResource
...
+                Xamarin.Forms.DependencyService.Get<ISave>().SaveAndView("WordToImage_" + i + ".jpeg", "image/jpeg", stream as MemoryStream);
+                i++;
+            }
+        }
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-to-Image-conversion/Specific-range-of-pages-Word-to-image).
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/word-to-image.md:334]: Add the below sentence in existing note

DocIO supports Word to image conversion in UWP application using DocIORenderer.

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/word-to-image.md (hunk 1 above)
Comment (Suriya-Balamurugan) [reply File-Formats/DocIO/word-to-image.md]: Ensured the code snippet using DocIORenderer in UWP application and added

File: File-Formats/DocIO/word-to-image.md (hunk 2 above)
Comment (Suriya-Balamurugan) [reply File-Formats/DocIO/word-to-image.md:506]: Addressed

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**